index.html

```diff
@@ -284,7 +284,31 @@ <h4 class="refer-link">LET'S CONNECT</h4>
                     </ul>
                 </div>
             </div>
-        </section>       
+        </section>  
+        <section id="contact-container">
+            <div id="contact-items">
+                <div id="description-contact-box">
+                    <h1 class="contact-title-1">Contact me</h1> 
+                    <p class="contact-description-1">
+                        If you have an application you are interested in developing, a
+                        feature that you need built or a project that needs coding.I’d
+                        love to help with it.
+                    </p>
+                </div>
+                <form id="input-contact-box" action="https://formspree.io/f/xpzbqlab" method="post">
+                    <label class="label" for="username"></label>
+                    <input id="username" class="input-i" name="username" type="text" maxlength="30"
+                        placeholder="Enobong" required="required" />
+                    <label class="label" for="email"></label>
+                    <input id="email" class="input-i" name="email" type="email" value="" placeholder="misong4real@yahoo.com"
+                        required="required" />
+                    <label class="label" id="input-message" for="msg"></label>
+                    <textarea class="input-m" name="msg" id="msg" maxlength="500" cols="20" rows="10"
+                        placeholder="Write your message here" required="required"></textarea>
```

Comment:
- [ ] Why don't you use `required` instead of `required="required"` to keep your code clean? :smile: